.github/workflows/post-release-changes.yml

```diff
@@ -0,0 +1,64 @@
+name: Post-Release Preparation.
+
+on:
+  workflow_call:
+    inputs:
+      REPO_URL:
+        description: '<github-account-name>/<repo-name>'
+        required: true
+        type: string
+      REPO_BRANCH:
+        required: true
+        type: string
+      BASE:
+        description: 'base branch for PR'
+        required: true
+        type: string
+    secrets:
+      SLACK_WEBHOOK_URL:
+        required: true
+      ACTION_PAT:
+        required: true
+
+jobs:
+  maven-post-release-preparation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.REPO_URL }}
+          ref: ${{ inputs.REPO_BRANCH }}
+      - name: Setup branch and env
+        run: |
+          # Strip git ref prefix from version
+          echo "BRANCH_NAME=$(echo ${{ github.ref }} | sed -e 's,.*/\(.*\),\1,')" >> $GITHUB_ENV
+          echo "GPG_TTY=$(tty)" >> $GITHUB_ENV
+
+      - name: Install xmlstartlet and xmllint
+        run: |
+          sudo apt-get update
+          sudo apt-get install xmlstarlet libxml2-utils
```

Comment:
Package installation is not required in this case

Reply:
removed package installation commands